### PR TITLE
Bug 1335689 - Fix Top Tabs crash where reloadData is called when view is not yet created.

### PR DESCRIPTION
**Pull request by farhanpatel**
The never ending bugs 😢

### CODE REVIEW

File: Client/Frontend/Browser/TopTabsViewController.swift [hunk 1]
```diff
@@ -107,9 +107,9 @@ class TopTabsViewController: UIViewController {
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
-    
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
         self.reloadData()
```
Comment (sleroux) [inline Client/Frontend/Browser/TopTabsViewController.swift]: I find it odd that the view isn't created when `viewWillAppear` is called. Moving the reload to `viewDidAppear` seems to late. Would calling `reloadData` from `viewDidLayoutSubviews` work better here?

Comment (farhanpatel) [inline Client/Frontend/Browser/TopTabsViewController.swift]: No. Unfortunately, this bug isn't just about calling reloadData before the view is created. It's more about other parts of the app that might add tabs after the TopTabs is created but before its displayed. For example the new tab that is created on first run.

Comment (farhanpatel) [inline Client/Frontend/Browser/TopTabsViewController.swift]: I guess I should have used a better title :(

**Comment by farhanpatel** [comment]
### SwiftLint found issues

#### Errors

File | Line | Reason |
| --- | ----- | ----- |
TopTabsLayout.swift | 85 | Opening braces should be preceded by a single space and on the same line as the declaration. 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Pull request merged**